gatsby-node.js

```diff
@@ -5,6 +5,25 @@ const TYPESENSE_ATTRIBUTE_NAME = "data-typesense-field"
 
 let utils = require("./lib/utils")
 
+function typesenseGetValue(fieldDefinition, attributeValue) {
```

Comment:
This function name reads a little strange IMO. What do you think about something like `typeCastValue` to make the intention explicit?

Reply:
Sounds good @jasonbosco I've updated the function name to your suggestion